packages/actor-init-sparql/test/ActorInitSparql-test.ts

```diff
@@ -475,6 +476,18 @@ describe('ActorInitSparql', () => {
         });
     });
 
+    it('should run with a tagged hypermedia and credentials in url and query file option from argv', () => {
```

Comment:
We want to make sure that the resulting context contains the correct credentials and source. Something like in this test: https://github.com/comunica/comunica/blob/d5edecea04f6b25d4235bea09dd6b3b7a1b8a12e/packages/actor-init-sparql/test/ActorInitSparql-test.ts#L759